examples/lang-err-integration-tests/call-builder/lib.rs

```diff
@@ -205,14 +207,7 @@ mod call_builder {
                     None,
                 )
                 .await
-                .expect("Client failed to call `call_builder::call_instantiate`.")
-                .value
-                .expect("Dispatching `call_builder::call_instantiate` failed.");
-
-            assert!(
-                call_result.is_some(),
```

Comment:
Should this check now be retained but changed to `call_result.value.is_none()` now that it returns `None` on success with no `LangError`

Reply:
Ah shit 🙈